components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java

```diff
@@ -1054,7 +1054,10 @@ public static String getApplicationName(ServiceProvider serviceProvider) {
      * @param userName     Username of the initiator.
      * @param tenantDomain Tenant domain of the initiator.
      * @return User id of the initiator.
+     * @deprecated
+     * This method is moved to IdentityUtil class as this will be used from other components as well.
      */
+    @Deprecated
     public static String getInitiatorId(String userName, String tenantDomain) {
```

Comment:
to avoid any code duplication, you can remove the logic in the method body and directly refer to the new method.